articles/django-crud-api/index.md

```diff
@@ -0,0 +1,292 @@
+
+### Prerequisites
+
+- Basics of Django web framework
+- Basics of Python programming language.
+- Postman installed.
+- Code editor of your choice installed.
+- Python and Virtualenv installed.
+  
+To verify if Python is installed and configured correctly on your system, Open the terminal and type in the command `python3 --version` on Linux/Mac or `python --version` if you are on windows.
+
+```python
+$ python3 --version
+Python 3.8.5
+```
+To verify virtualenv installation execute the command `virtualenv --version` on the terminal.
+```python
+$ virtualenv --version
+virtualenv 20.0.35 from /home/user/.local/lib/python3.8/site-packages/virtualenv/__init__.py
+```
+
+### Creating the todo project.
+
+We will start by creating our project's work directory and a virtual environment for our project. The virtual environment makes it possible to run our project and its dependencies in an isolated environment.
+
+Run `mkdir ~/todo` to create our working directory.
+
+```python
+$ mkdir ~/todo
+$ cd todo
+```
+Run `virtualenv venv` to create a virtual environment for our project and `source venv/bin/activate` to activate the virtual environment for our project.
+To install Django and Django rest framework in our virtual environment we run the command `pip3 install django` and `pip3 install djangorestframework`.
+
+```python
+$ pip install django
+Collecting django
+  Using cached Django-3.1.3-py3-none-any.whl (7.8 MB)
+Collecting asgiref<4,>=3.2.10
+  Downloading asgiref-3.3.1-py3-none-any.whl (19 kB)
+Collecting pytz
+  Using cached pytz-2020.4-py2.py3-none-any.whl (509 kB)
+Collecting sqlparse>=0.2.2
+  Using cached sqlparse-0.4.1-py3-none-any.whl (42 kB)
+Installing collected packages: asgiref, pytz, sqlparse, django
+Successfully installed asgiref-3.3.1 django-3.1.3 pytz-2020.4 sqlparse-0.4.1
+```
+
+```python
+$ pip install djangorestframework
+Collecting djangorestframework
+  Downloading djangorestframework-3.12.2-py3-none-any.whl (957 kB)
+     |████████████████████████████████| 957 kB 595 kB/s 
+Requirement already satisfied: django>=2.2 in ./venv/lib/python3.8/site-packages (from djangorestframework) (3.1.3)
+Requirement already satisfied: asgiref<4,>=3.2.10 in ./venv/lib/python3.8/site-packages (from django>=2.2->djangorestframework) (3.3.1)
+Requirement already satisfied: sqlparse>=0.2.2 in ./venv/lib/python3.8/site-packages (from django>=2.2->djangorestframework) (0.4.1)
+Requirement already satisfied: pytz in ./venv/lib/python3.8/site-packages (from django>=2.2->djangorestframework) (2020.4)
+Installing collected packages: djangorestframework
+Successfully installed djangorestframework-3.12.2
+```
+
+Lets create out django-todo project now by running the command `django-admin startproject django_todo`
```

Comment:
Inconsistency in the project naming
"django-todo" and "django_todo" pick one and use it consistently

Also correct 
"Let's create our..."